<launch>
  <arg name="cam_topic"  />
  <arg name="param_file" />
  
  <!-- <node name="object_detector" pkg="ffld_ros" type="detect_object_node"
        output="screen" launch-prefix="gdb -ex run">
    <param name="overlap" value="0.4" /> -->    <!-- Minimum overlap in non maxima suppression (default 0.5)  -->
    <!-- <param name="cam_topic" value="$(arg cam_topic)" />
    <rosparam file="$(arg param_file)" />
  </node> -->

  <!-- <node name="object_detector_$(arg model)" pkg="ffld_ros" type="detect_object_node"
        output="screen" >
    <param name="overlap" value="0.4" />
    <param name="model" value="$(arg model)" />
    <param name="model_directory" value="$(model_directory)" />
  </node> -->

  <node name="object_detector" pkg="ffld_ros" type="object_detector_launcher_node" output="screen" >
    <param name="cam_topic" value="$(arg cam_topic)" />
    <rosparam file="$(arg param_file)" />
  </node>

  <node name="object_detection_visualizer" pkg="ffld_ros" type="object_detection_visualizer" output="screen" >
  	<rosparam file="$(arg param_file)" />
  </node>

</launch>

